flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -1760,39 +1760,31 @@ protected void cancelTask() {
      */
     @Test
     public void testThroughputSchedulerStartsOnInvoke() throws Exception {
-        // given: Throughput meter which finishes the task when calculateThroughput was called.
-        UnAvailableTestInputProcessor inputProcessor = new UnAvailableTestInputProcessor();
-        AtomicInteger desirableNumberOfCalculation = new AtomicInteger(2);
-        try (MockEnvironment mockEnvironment =
-                new MockEnvironmentBuilder()
-                        .setTaskConfiguration(
-                                new Configuration()
-                                        .set(BUFFER_DEBLOAT_PERIOD, Duration.ofMillis(1)))
+        CompletableFuture<?> finishFuture = new CompletableFuture<>();
+        try (StreamTaskMailboxTestHarness<String> harness =
+                new StreamTaskMailboxTestHarnessBuilder<>(OneInputStreamTask::new, STRING_TYPE_INFO)
+                        .modifyStreamConfig(
+                                config ->
+                                        config.getConfiguration()
+                                                .set(BUFFER_DEBLOAT_PERIOD, Duration.ofMillis(1)))
+                        .addInput(STRING_TYPE_INFO)
+                        .setupOutputForSingletonOperatorChain(
+                                new TestBoundedOneInputStreamOperator())
                         .setThroughputMeter(
```

Comment:
Oh, I forgot to rename the setter to `setThroughputCalculator`. I will need to not forgot to fix it.